core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -3764,6 +3865,30 @@ public Scope getScope() {
     return mScope;
   }
 
+  /**
+   * @return the displayType which indicates how the property value should be displayed
+   */
+  public DisplayType getDisplayType() {
+    return mDisplayType;
+  }
+
+  /**
+   * Gets the display value for the actual value.
+   * @param value actual value of the property
+   * @return display value of the property
+   */
+  public String getDisplayValue(String value) {
```

Comment:
- what if `value` is null?  Can we consolidate `alluxio.util.ConfigurationUtils.valueAsString` also here?
- I don't think this should be an instance method, considering to have this as a static method (maybe in `ConfigurationUtils`? Maybe you can improve upon on `ConfigurationUtils.valueAsString`?

Reply:
per our discussion, we leave this function and `alluxio.util.ConfigurationUtils.valueAsString` as separate functions given that there are places that might not want to apply both transformations.